swarm/storage/memstore.go

```diff
@@ -186,11 +186,9 @@ func (s *MemStore) Put(entry *Chunk) {
 			if entry.ReqC == nil {
 				entry.ReqC = node.entry.ReqC
 			}
-			// if node.entry.dbStoredC != nil {
-			// 	entry.dbStored = node.entry.dbStored
-			// 	entry.dbStoredC = node.entry.dbStoredC
-			// 	entry.dbStoredMu = node.entry.dbStoredMu
-			// }
+			entry.dbStored = node.entry.dbStored
```

Comment:
lets please get rid of this lock and bool. it was a measure to handle a bug. 
we should not write in a batch twice in dbstore thats all